Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInSignUp.kt

```diff
@@ -261,15 +256,12 @@ fun OrSignInAsGuest(
         modifier = modifier,
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
-        Surface {
-            CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
-                Text(
-                    text = stringResource(id = R.string.or),
-                    style = MaterialTheme.typography.subtitle2,
-                    modifier = Modifier.paddingFromBaseline(top = 25.dp)
-                )
-            }
-        }
+        Text(
```

Comment:
Is there a reason there was a `Surface` wrapping this before? Is it no longer needed?

Reply:
I couldn't see any reason for it since this whole Composable gets used only where the background is already matching surface. Maybe it was necessary in an earlier version and wasn't cleaned up previously.